packages/vaadin-lumo-styles/utilities/border.js

```diff
@@ -91,6 +91,16 @@ export const border = css`
     border-color: var(--lumo-success-color-10pct);
   }
 
+  .border-warning {
+    border-color: var(--lumo-warning-color);
+  }
+  .border-warning-10 {
+    border-color: var(--lumo-warning-color-10pct);
+  }
+  .border-warning-text {
+    border-color: var(--lumo-warning-text-color);
+  }
```

Comment:
This feels out of place.

Reply:
The name, or the existence of a text-colored border?

If the former, I agree that the name is... sub-optimal. Perhaps `.border-warning-strong` instead?

Reply:
Yeah, the name makes it feel out of place. I dunno. I kinda throw my hands up here and let you decide what to do 😅

Reply:
Yeah no, agree. Renaming to `.border-warning-strong`.